pricenode/install_pricenode_debian.sh

```diff
@@ -34,6 +34,11 @@ echo "[*] Upgrading apt packages"
 sudo -H -i -u "${ROOT_USER}" DEBIAN_FRONTEND=noninteractive apt-get update -q
 sudo -H -i -u "${ROOT_USER}" DEBIAN_FRONTEND=noninteractive apt-get upgrade -qq -y
 
+echo "[*] Installing Git LFS"
+sudo -H -i -u "${ROOT_USER}" curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
```

Comment:
Why is this needed? Doesn't ubuntu have a git-lfs package by default?

Reply:
I also think ```apt install git-lfs``` should work just fine. No need for external packages.

Reply:
This is used to ensure the latest git-lfs package is installed. I don't recall now, but perhaps the latest isn't necessary for our needs and it should be fine to remove this line.
https://github.com/git-lfs/git-lfs/wiki/Installation#debian-and-ubuntu